scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
```

Comment:
This is an example of a trade off where we do a just a bit more work, but win in cross-project consistency where every developer here is able to immediately know where to go to run key project functions. And external developers are able to do the same because we document the scripts in the README. 

In either case, I'm fine with deferring this to #161.